src/WebJobs.Script/Workers/Profiles/IWorkerProfileManager.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.Azure.WebJobs.Script.Workers.Profiles;
+using Microsoft.Azure.WebJobs.Script.Workers.Rpc;
+
+namespace Microsoft.Azure.WebJobs.Script.Workers
+{
+    public interface IWorkerProfileManager
```

Comment:
does this also need to be internal?

Reply:
No, it has to be public as WebHostRpcWorkerChannelManager needs access to this